api/docs/release.dox

```diff
@@ -138,6 +138,8 @@ changes:
    via struct.value.reg_and_element_size.element_size.
```

Comment:
A key missing piece here is that there are no added tests: please add regression tests.  Since this interacts with windows we would want tests with and without windows.

Reply:
Yes, I will add tests. Currently multi-threading also is not handled correctly.

Reply:
Added some tests.